modules/alphamat/include/opencv2/alphamat.hpp

```diff
@@ -9,19 +9,28 @@
 
 /**
  * @defgroup alphamat Alpha Matting
- * This module is dedicated to compute alpha matting of images, given the input image and an input trimap.
- * The samples directory includes easy examples of how to use the module.
+ * Alpha matting is used to extract a foreground object with soft boundaries from a background image.
+ *
+ * This module is dedicated to computing alpha matte of objects in images from a given input image and a greyscale trimap image that contains information about the foreground, background and unknown pixels. The unknown pixels are assumed to be a combination of foreground and background pixels. The algorithm uses a combination of multiple carefully defined pixels affinities to estimate the opacity of the foreground pixels in the unkown region.
+ *
+ * The implementation is based on "Designing Effective Inter-Pixel Information Flow for Natural Image Matting" by Yağız Aksoy, Tunç Ozan Aydın and Marc Pollefeys, CVPR 2017.
```

Comment:
Please avoid non-ASCII symbols in public headers. For references consider using `@cite` and entry in .bib file.

Reply:
Done